samples/sample_hevc_fei_abr/include/pipeline_hevc_fei.h

```diff
@@ -72,6 +72,10 @@ class EncoderContext
     {
         return m_encoder->SubmitFrame(task);
     }
+    void SetNotifyErrorCallback(IEncoder::NotifyErrorCallback callback)
+    {
+        return m_encoder->SetNotifyErrorCallback(callback);
```

Comment:
it's void type return, so just don't return anything

Reply:
Okay